buildkite/src/gen/Jobs.dhall

```diff
@@ -1,3 +1,74 @@
--- This file is autogenerated during builds. It remains checked in to ensure
--- dhall configuration can still execute locally without running codegen.
-let Pipeline = ../Pipeline/Dsl.dhall in [] : List Pipeline.CompoundType
+[ -- Autogenerated do not edit by hand ,
```

Comment:
Are these still in the scope of this PR? I thought it's only for a test for archive nodes.